Board.php

```diff
@@ -0,0 +1,277 @@
+<?php
+
+namespace GameOfLife;
+
+use GameOfLife\outputs\BaseOutput;
+
+/**
+ * Create playing board.
+ * The class Field is responsible für generating and printing the fields and cells needed for the GoL.
+ * It will also count the cell's neighbours and generate the next generation of cells by following the default GoL rules.
+ * In the end, the class "start" will execute the code and start the GameOfLife.
+ * @package gameoflife
+ */
+class Board
+{
+    protected $board = [];
+    protected $width;
+    protected $height;
+//    private $history = [];
+//    private $generationCount = 0;
+
+    /**
+     * Board constructor
+     * @param $_width int
+     * @param $_height int
+     */
+    public function __construct($_width, $_height)
+    {
+        $this->width = $_width;
+        $this->height = $_height;
+
+        $this->constructBoard();
+    }
+
+    /**
+     * Create board from Array.
+     * @param $_array array
+     * @return board
+     */
+    public static function createFromArray($_array)
+    {
+        $height = count($_array);
+        $width = count($_array[0]);
+
+        $board = new Board($width, $height);
+
+        for($y = 0; $y < $height; $y++)
+        {
+            for($x = 0; $x < $width; $x++)
+            {
+                $board->setBoardValue($x, $y, $_array[$y][$x]);
+            }
+        }
+        return $board;
+    }
+
+    /**
+     * Construct empty board
+     */
+    public function constructBoard()
+    {
+        for ($y = 0; $y < $this->height; $y++)
+        {
+            for ($x = 0; $x < $this->width; $x++)
+            {
+                $this->board[$x][$y] = new Field($this, $x, $y);
+            }
+        }
+    }
+
+    /**
+     * Set board's cells
+     * @param $_x int for the x-Position of the cells
+     * @param $_y int for the y-position of the cells
+     * @param $_state bool for the status of cells (dead or alive)
+     */
+    public function setBoardValue($_x, $_y, $_state)
```

Comment:
Replacing every setBoardValue() would require a LOT of work, because I am using this function in almost every unit-test. I would like to avoid that and stick with this function. Also the "silent fail", if I have understood correctly, is intendet.

Reply:
Is ok for me.